packages/components/src/components/post-accordion-item/post-accordion-item.scss

```diff
@@ -0,0 +1,22 @@
+@use '@swisspost/design-system-styles/components/accordion';
+@use '@swisspost/design-system-styles/mixins/accordion' as accordion-mx;
+
+:host {
+  display: block;
+}
+
+.accordion-button {
+  cursor: pointer;
+
+  > ::slotted(.text-truncate) {
+    display: block;
+  }
+}
+
+post-accordion-item::part(accordion-item) {
+  @include accordion-mx.background-color;
+
+  post-accordion-item + & {
```

Comment:
```suggestion
  & + post-accordion-item {
```

Reply:
I want `post-accordion-item + post-accordion-item::part(accordion-item)` not `post-accordion-item::part(accordion-item) + post-accordion-item`, it is the `accordion-item` part that should not have a border.